src/lib/cozy-client/slices/sharings.js

```diff
@@ -386,7 +397,7 @@ const getDocumentActiveSharings = (state, doctype, id) => {
     )
   ]
     .map(p => getSharing(state, p.attributes.source_id))
-    .filter(s => !s.attributes.revoked)
+    .filter(s => s && s.attributes && !s.attributes.revoked)
```

Comment:
Here you check for `s`, but in the previous fixes you are not checking `p` and `s`. Is there a reason for that? (also, booo single letter variables.)

Reply:
ouais, je suis tombé sur une erreur de `s is undefined` à ce niveau-là... Mais bon, ici, rien de bloquant dans le runtime, donc si on en a oublié un, c'est pas bien grave.